epoxy-model/src/main/java/com/airbnb/epoxy/EpoxyClass.java

```diff
@@ -0,0 +1,16 @@
+
+package com.airbnb.epoxy;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Used to annotate {@link EpoxyModel} classes in order to generate a
+ * subclass of that model with getters, setters, equals, and hashcode for the annotated fields.
+ */
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.CLASS)
+public @interface EpoxyClass {
```

Comment:
We could maybe go with `EpoxyModelClass` to be more clear, since any class in this library could be considered an Epoxy class